ledger/src/blockstore.rs

```diff
@@ -2026,6 +2021,22 @@ impl Blockstore {
                     (0, 0)
                 }
             };
+
+            let (min_fec_set_count, last_fec_set_count) = if num_repaired == 0 {
+                self.turbine_fec_set_stats
+                    .lock()
+                    .unwrap()
+                    .get_min_index_count(&slot)
+            } else {
+                // remove metadata for repaired slots
+                let mut stats = self.turbine_fec_set_stats.lock().unwrap();
+                let min_counts = stats.get_min_index_count(&slot);
+                stats.remove(&slot);
+                min_counts
+            };
+            let last_fec_set_size =
+                slot_meta.last_index.unwrap_or(0) % MAX_DATA_SHREDS_PER_FEC_BLOCK as u64;
```

Comment:
This number is number of data shreds in the last batch.
The batch-sizes (or fec_set_size) is alwasy 2 * MAX_DATA_SHREDS_PER_FEC_BLOCK.

Reply:
I was breaking this out as the number of data shreds in the last batch is probably < MAX_DATA_SHREDS_PER_FEC_BLOCK and could trigger is_full with less shreds in the batch. If you don't think this is useful I can remove the special case.